<launch>
  
  <node name="mpc_server" pkg="mobile_mpc" type="mpc_server" respawn="false" output="screen"/>
  <!--node name="errorComputer" pkg="mobile_mpc" type="errorComputer" /-->

  <!--node name="rviz" type="rviz" pkg="rviz"
    args="-d $(find mobile_mpc)/config/mpc_planner.rviz"/-->

</launch>
